ontopy/utils.py

```diff
@@ -835,3 +835,34 @@ def recur(o):
                     )
 
     return layout
+
+
+def copy_annotation(onto, src, dst):
+    """In all classes and properties in `onto`, copy annotation `src` to `dst`.
+
+    The `src` and `dst` can either be provided as a label string or a full IRI.
+    """
+    if onto.world[src]:
+        src = onto.world[src]
+    else:
+        src = onto[src]
+
+    if onto.world[dst]:
+        dst = onto.world[dst]
+    elif dst in onto:
+        dst = onto[dst]
+    else:
+        if "://" not in dst:
+            raise ValueError(
+                "new destination annotation property must be provided as "
+                "a full IRI"
+            )
+        name = min(dst.rsplit("#")[-1], dst.rsplit("/")[-1])
+        iri = dst
+        dst = onto.new_annotation_property(name, owlready2.AnnotationProperty)
+        dst.iri = iri
```

Comment:
Needs test

Reply:
This is already tested in tests/tools/test_ontoconvert.py

Just codecov that doesn't understand tests done in subprocesses...